src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/writetag/InputWriteTagEmptyTag.java

```diff
@@ -9,14 +9,14 @@
 */
 
 package com.puppycrawl.tools.checkstyle.checks.javadoc.writetag;
-
+// violation 7 lines below , 'Javadoc .*'
```

Comment:
Please put more text to message to let us fully understand violation.
We use .* Only to comply to line limit 100

Reply:
Done @romani